workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html

```diff
@@ -13,8 +14,24 @@
 
 [#assign message = "While immediate callbacks are reserved for those with platinum level, a support member will get in touch within 1 business day during our <a href='https://web.liferay.com/support/coverage-areas-and-hours'>standard business hours</a> of your designated Support Center. In the meantime, consider creating a <a href='https://help.liferay.com/hc'>support ticket</a> which can help expedite a resolution." /]
 
+[#if languageId == "es_ES"]
```

Comment:
@JoMaAlves can we convert this to a traditional language key in liferay, I am not sure if we can define those in the site initializer or not, but would be cleaner to just use a language key and set it up in the oob language portlet

Reply:
I'll take a look

Reply:
Done!